scripts/scripts.js

```diff
@@ -1424,7 +1430,35 @@ sampleRUM.always.on('convert', (data) => {
         }
       };
     }
-    console.debug('push to datalayer', evtDataLayer);
-    window.digitalData.push(evtDataLayer);
+
+    //Check if a Form Complete event has fired - and if the conversion value (data.target) or form name (data.source) is defined
+    if (evtDataLayer.event === "Form Complete" && (data.target === undefined || data.source === undefined)) {
+      //if a buffer has already been set and tempEvtDataLayer exists, merge the two evt objects to send to the data layer
+      if (bufferTimeoutId !== undefined && tempEvtDataLayer !== undefined) {
+        evtDataLayer = { ...evtDataLayer, ...tempEvtDataLayer }
+      }
+      else {
+        //temporarily hold the evtDataLayer object until the timeout is complete
+        tempEvtDataLayer = evtDataLayer;
```

Comment:
You are just referencing here, and aren't creating a copy. Given that you don't access `tmpEventDataLayer` in the remainder of the block, this does nothing at all and you can just check for the existence of `bufferTimoutId` above.